workday.sgnl.yaml

```diff
@@ -0,0 +1,92 @@
+description: "Workday as a System of Record"
+address: "wd2-impl-services1.workday.com" 
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: Workday-1.0.0
+adapterConfig: ewogICAgImFwaVZlcnNpb24iOiAidjEiLAogICAgIm9yZ2FuaXphdGlvbklkIjogInt7SW5wdXQgUmVxdWlyZWR9fSIKfQ==
+auth:
+  - bearer:
+      authToken: "Bearer {{Input Required}}"
+entities:
+  Worker:
+    displayName: Worker
+    externalId: allWorkers # Name of table in Workday
+    description: Worker entity in Workday
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workerId
+        externalId: $.worker.id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: workerDescriptor
```

Comment:
If this is the email of the Worker, index this attribute

Reply:
I indexed the WorkEmail.descriptor which is the email of the worker.